src/index.html

```diff
@@ -0,0 +1,69 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="css/vote-CSS.css" />
```

Comment:
So try not to have the word `CSS` in the name of the file, the file type tells you that. If you just have one CSS file, the normal convention is something like `main.css` or `styles.css` 👍

Reply:
Altered to styles.css